load_tests/Makefile

```diff
@@ -0,0 +1,44 @@
+.PHONY: init install lint format test locust-rest
+
+RASA_VERSION=1.9.4
+
+UID:=$(shell id -u)
+GID:=$(shell id -g)
+
+help:
+	@echo "    init"
+	@echo "        Initialize virtual environment"
+	@echo "    install"
+	@echo "        Install dependencies"
+	@echo "    lint"
+	@echo "        Check style with pylama, mypy and black"
+	@echo "    format"
+	@echo "        Format code with black"
+	@echo "    test"
+	@echo "        Run py.test (use TEST_FILE variable to test a single file)"
+	@echo "    locust-rest"
+	@echo "        Execute locust file, starting the locust web server"
+
+init:
+	curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+install:
+	poetry install
+
+lint:
+	poetry run pylama
+	poetry run mypy .
+	poetry run black --check .
+	poetry run isort --check
+
+format:
+	poetry run black .
+	poetry run isort
+
+test:
+	poetry run py.test \
+		--cov-report term-missing:skip-covered \
+		--cov-report html
+
+locust-rasa:
+	LOAD_TEST_TARGET_URL=http://localhost:5005/ poetry run locust -f load_tests/socketio_locustfile.py --no-web -c 5 -r 5 --run-time 1200s
```

Comment:
Seems kinda weird to have this env variable here

Reply:
Removed it since the user should source the env variables himself.